### PR TITLE
test: test \n in SQL column comments

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
no issue, just add a test

### Check List 

Tests 

 - Integration test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: tests/online_ddl/run.sh [hunk 1]
```diff
@@ -27,8 +27,11 @@ function real_run() {
 	sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker2/relay_log" $WORK_DIR/source2.yaml
 	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
 
+  export GO_FAILPOINTS="github.com/pingcap/dm/syncer/ExitAfterSaveOnlineDDL=return()"
```
Comment (Ehco1996) [inline tests/online_ddl/run.sh:30]: add this failpoint for ?

Comment (lance6716) [reply tests/online_ddl/run.sh:30]: I want to test there's no bug loading checkpoint of online DDL, so after save an online DDL checkpoint, panic and restart the worker.

**Comment by GMHDBJD** [comment]
Seems need make fmt in my local environment, but ci pass. 🤔

**Comment by lance6716** [comment]
> Seems need make fmt in my local environment, but ci pass. 🤔

will check and add fmt back in this PR.

**Comment by GMHDBJD** [comment]
Seems because ci remove the build check now.

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**